### PR TITLE
latest release notes additions, Jan 20 2020

### PR DESCRIPTION
**Pull request by sheriff-rh**
Added the remaining bug fixes, found and fixed a typo.

### CODE REVIEW

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -656,6 +656,13 @@ Podman containers on CRI-O restore so that they are no longer deleted from
 storage upon startup.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1758500[*BZ#1758500*])
 
+*Developer Console*
+
+* An invalid property was introduced into the Pipeline Operator during an upgrade.
+As a result, Pipelines could no longer start from the UI. The  property has been
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: extra space before property?
s/has been/is now

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -768,6 +839,18 @@ result, the `/metrics` endpoint now allows metrics to flow to Prometheus without
 causing a 500 error.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1779285[*BZ#1779285*])
 
+* Upstream changes were made to the long path names test. Pods with names longer
+than 255 character were not logged and no warning was issues. Now, the long
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: s/issues/issued

File: release_notes/ocp-4-3-release-notes.adoc [hunk 3]
```diff
@@ -768,6 +839,18 @@ result, the `/metrics` endpoint now allows metrics to flow to Prometheus without
...
+names test has been removed and pods with names longer than 255 characters will
+log as expected.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: s/has been/is
s/pods/Pods

File: release_notes/ocp-4-3-release-notes.adoc [hunk 4]
```diff
@@ -758,6 +796,39 @@ objects are cleaned up properly.
 are no longer dropped.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1747532[*BZ#1747532*])
 
+* Previously, in clusters that had high network traffic volumes with packet loss,
+a once-successful connection to a service could fail with a "Connection reset by
+peer" error. As a result, clients had to reconnect and retransmit. An update has
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: s/"Connection reset by peer"/*Connection reset by peer*

**Comment by ahardin-rh** [comment]
Just a few minor comments from me. Otherwise, looks great!

**Comment by sheriff-rh** [comment]
All suggestions have been corrected; merging.

**Pull request merged**